.github/workflows/build.yaml

```diff
@@ -23,6 +23,11 @@ jobs:
       with:
         path: shadow
         repository: shadow/shadow
+    # Run cmake to generate shd-config.h
+    - name: Run cmake
+      run: |
+        sudo apt-get install cmake
+        cmake -S . -B build
```

Comment:
I think you would need to cd into the `shadow` directory first?

Reply:
thanks, fixed